src/environmentd/src/http.rs

```diff
@@ -452,12 +452,28 @@ impl AuthedClient {
         adapter_client: &Client,
         user: AuthedUser,
         active_connection_count: SharedConnectionCounter,
+        options: BTreeMap<String, String>,
     ) -> Result<Self, AdapterError> {
         let AuthedUser(user) = user;
         let drop_connection = DropConnection::new_connection(&user, active_connection_count)?;
         let conn_id = adapter_client.new_conn_id()?;
-        let session = adapter_client.new_session(conn_id, user);
-        let (adapter_client, _) = adapter_client.startup(session, vec![]).await?;
+        let mut session = adapter_client.new_session(conn_id, user);
+        let mut set_setting_keys = Vec::new();
+        for (key, val) in options {
+            const LOCAL: bool = false;
```

Comment:
I'm slightly afraid to ask, but what is the purpose of this, vs just passing `false` below?

Reply:
I don't know! I just moved this from somewhere else.

Reply:
My guess though is it's for documentation, so you don't have to go look up the definition of the called function to see the name of the parameter.